assets/js/modules/reader-revenue-manager/components/dashboard/RRMSetupSuccessSubtleNotification.js

```diff
@@ -144,11 +170,37 @@ export default function RRMSetupSuccessSubtleNotification( {
 	// Sync publication data when user re-focuses window.
 	useRefocus( syncPublication, 15000 );
 
+	const showingSuccessNotification =
+		notification === 'authentication_success' &&
+		slug === READER_REVENUE_MANAGER_MODULE_SLUG;
+
+	// On successful module setup, if the payment option is not set,
+	// show the publication approved overlay notification.
+	useEffect( () => {
+		if (
+			isRRMV2Enabled &&
+			showingSuccessNotification &&
+			publicationOnboardingState === ONBOARDING_COMPLETE &&
+			paymentOption === ''
+		) {
+			setValue(
+				UI_KEY_READER_REVENUE_MANAGER_SHOW_PUBLICATION_APPROVED_NOTIFICATION,
+				true
+			);
```

Comment:
As per my comment above, I think it would be easier to follow if we dismissed this notification here:

```suggestion
			);

			dismissNotice();
```

Reply:
Updated, thank you!